docs/index.rst

```diff
@@ -134,9 +134,15 @@ which fills in the version number.
    tutorials.rst
    examples.rst
    scripts/scripts
-   visualization/visualization
    rst/files
 
+.. toctree::
+   :maxdepth: 4
+   :caption: Visualization
+   :glob:
+   
+   visualization/*/index
```

Comment:
be explicit ;)

Reply:
Why do you want to be explicit here? It seems intuitive to do it in this way, since you can add extra sections by just adding directories to the `visualization` directory. I.e. you make sure the documentation tree follows the filesystem tree and you don't need to modify the main page.

I think it's worse to be explicit since you can then have an unordered file structure that works (but it can be very confusing, specially when you scale it), isn't it?

Reply:
Sorry for the late return.
I will change this to be explicit. Better to control the order of content, than relying on file-systems. :)

Reply:
Ok :+1: